rpcs3/Emu/System.cpp

```diff
@@ -385,55 +396,61 @@ void Emulator::Init(bool add_only)
 
 	if (g_cfg.vfs.init_dirs)
 	{
-		make_path_verbose(dev_bdvd);
-		make_path_verbose(dev_hdd0);
-		make_path_verbose(dev_hdd1);
-		make_path_verbose(dev_flash);
-		make_path_verbose(dev_flash2);
-		make_path_verbose(dev_flash3);
-		make_path_verbose(dev_usb);
-		make_path_verbose(dev_hdd0 + "game/");
-		make_path_verbose(dev_hdd0 + reinterpret_cast<const char*>(u8"game/＄locks/"));
-		make_path_verbose(dev_hdd0 + "home/");
-		make_path_verbose(dev_hdd0 + "home/" + m_usr + "/");
-		make_path_verbose(dev_hdd0 + "home/" + m_usr + "/exdata/");
-		make_path_verbose(save_path);
-		make_path_verbose(dev_hdd0 + "home/" + m_usr + "/trophy/");
+		make_path_verbose(dev_bdvd, true);
+		make_path_verbose(dev_flash, true);
+		make_path_verbose(dev_flash2, true);
+		make_path_verbose(dev_flash3, true);
+		make_path_verbose(dev_usb, true);
 
-		if (!fs::write_file(user_path, fs::create + fs::excl + fs::write, "User"s))
+		if (make_path_verbose(dev_hdd1, true))
 		{
-			if (fs::g_tls_error != fs::error::exist)
+			make_path_verbose(dev_hdd1 + "caches/", true);
+		}
+
+		if (make_path_verbose(dev_hdd0, true))
+		{
+			make_path_verbose(dev_hdd0 + "game/", true);
+			make_path_verbose(dev_hdd0 + reinterpret_cast<const char*>(u8"game/＄locks/"), true);
+			make_path_verbose(dev_hdd0 + "home/", true);
+			make_path_verbose(dev_hdd0 + "home/" + m_usr + "/", true);
+			make_path_verbose(dev_hdd0 + "home/" + m_usr + "/exdata/", true);
+			make_path_verbose(save_path, true);
+			make_path_verbose(dev_hdd0 + "home/" + m_usr + "/trophy/", true);
+
+			if (!fs::write_file(user_path, fs::create + fs::excl + fs::write, "User"s, true))
```

Comment:
I think you added a true here by mistake

Reply:
Fixed,